avocado/utils/pci.py

```diff
@@ -97,20 +98,25 @@ def get_disks_in_pci_address(pci_address):
     return disk_list
 
 
-def get_nics_in_pci_address(pci_address):
+def get_interface_in_pci_address(pci_address, pci_class):
```

Comment:
Please keep the old `get_nics_in_pci_address` function pointing to `get_interface_in_pci_address(pci_address, "net")`

also I'd suggest using name `get_interfaces_in_pci` or `get_interfaces_in_pci_address` as we are looking for one or many interfaces...